testing/xfpga/test_sysfs_c.cpp

```diff
@@ -186,11 +193,17 @@ TEST_P(sysfsinit_c_p, sysfs_initialize) {
   for (const auto &d : platform_.devices) {
     devices[to_uint32(d.segment, d.bus, d.device, d.function)] = d;
   }
-
+  if (platform_.devices[0].num_vfs) {
+    auto d = platform_.devices[0];
+    d.function = 1;
+    d.device_id++;
```

Comment:
Not convinced this will always be true.

Reply:
You're right. This will work for now but fixing this the right way would require more work to the testing hw tables and data structures.